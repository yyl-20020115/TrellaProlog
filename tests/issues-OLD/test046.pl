:-initialization(main).

main :-
	1 =.. L1, write(L1), nl,
	aa =.. L2, write(L2), nl,
	[aa] =.. L3, write(L3), nl,
	[aa,bb] =.. L4, write(L4), nl,
	[aa,bb,cc] =.. L5, write(L5), nl.
