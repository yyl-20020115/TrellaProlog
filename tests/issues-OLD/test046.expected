[1]
[aa]
[.,aa,[]]
[.,aa,[bb]]
[.,aa,[bb,cc]]
